pumps/resurface.go

```diff
@@ -74,10 +79,26 @@ func (rp *ResurfacePump) Init(config interface{}) error {
 		rp.log.Info(rp.GetName() + " Initialized (Logger disabled)")
 		return errors.New("logger is not enabled")
 	}
+	rp.initWorker()
 	rp.log.Info(rp.GetName() + " Initialized")
 	return nil
 }
 
+func (rp *ResurfacePump) initWorker() {
+	rp.data = make(chan []interface{}, 5)
```

Comment:
Ideally, it would be 1 instead of 5. After some load testing I found that having a couple extra slots to avoid blocking right away wouldn't hurt (plus some more just to be safe). I do agree it would be better to make it configurable by defining a new parameter for this pump backend. Please, let me know if this is something you need and I can make a new PR with the required changes. Thanks!